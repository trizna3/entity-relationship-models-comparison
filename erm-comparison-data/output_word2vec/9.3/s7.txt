students-students
contracts-contracts
buildings-buildings
payments-payments
rooms-rooms
account-account
EXTRACT_ATTR_TO_OWN_ENTITY_SET:S-account-students
