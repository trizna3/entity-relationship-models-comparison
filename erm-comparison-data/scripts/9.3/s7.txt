entity set hostel {
	"name"
};

entity set buildings {
	"floor count"
	"address"
};

entity set rooms {
	"capacity"
	"fee"
};

entity set contracts {
	"start"
	"end"
};

entity set students {
	"name"
	"account"
};

entity set payments {
	"fee"
	"when"
};

association between 
{
	hostel ONE 
	buildings MANY
}{};

association between 
{
	buildings ONE 
	rooms MANY
}{};

association between 
{
	rooms ONE 
	contracts ONE
}{};

association between 
{
	contracts ONE 
	payments MANY
}{};

association between 
{
	payments ONE 
	students MANY
}{};

association "contract state" between 
{
	contracts ONE 
	students MANY
}{
	"new"
	"old"
};
