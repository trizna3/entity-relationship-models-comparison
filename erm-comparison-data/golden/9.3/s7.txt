buildings-buildings
rooms-rooms
contracts-contracts
payments-payments
students;account-students
CONTRACT_11_ASSOCIATION:E-students-account
DECOMPOSE_ATTRIBUTE:S-address-buildings